packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -10,6 +10,12 @@ import * as classNames from "classnames";
 import * as PureRender from "pure-render-decorator";
 import * as React from "react";
 
+export enum TruncatedPopover {
```

Comment:
I'd prefer a clearer noun here; the `core` package has conditioned me to expect any `Popover` to be a React component. Maybe something like `TruncatedPopoverScheme` or `TruncationScheme`?